rest_framework-stubs/generics.pyi

```diff
@@ -37,6 +41,9 @@ class GenericAPIView(views.APIView, UsesQuerySet[_MT_co]):
     lookup_url_kwarg: str | None
     filter_backends: Sequence[type[BaseFilterBackend | BaseFilterProtocol[_MT_co]]]
     pagination_class: type[BasePagination] | None
+    if sys.version_info >= (3, 9):
```

Comment:
Well, all these methods actually return `cls`: https://github.com/search?q=repo%3Aencode%2Fdjango-rest-framework%20__class_getitem__&type=code

Reply:
Should i be `-> type[Self]` then?

Reply:
I appreciate you taking the time to inform me. Thank you!!